src/components/Line/style.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+import { LineProps } from './type'
+
+const StyledLine = styled.div<LineProps>`
+  width: 100%;
+  max-width: ${(props: LineProps) => props.maxWidth};
+  height: ${(props) => (props.dashed ? '1px' : '1px')};
```

Comment:
do we need this prop `dashed` ?

Reply:
not necessary just an option if necessary